### ISSUE TITLE
Deepvariant>0.6.1 no longer installable

### ISSUE THREAD

**Issue by votti** [bioconda/bioconda-recipes#44200]
Currently the highest `deepvariant` version I still manage to install is `0.6.1` which is severely outdated.

The error when trying:
```
mamba create -n deepvariant -c bioconda -c conda-forge  deepvariant=1.5 
```
is
```
The following package could not be installed
└─ deepvariant 1.5**  is uninstallable because it requires
   └─ tensorflow-estimator 2.0.* , which does not exist (perhaps a missing channel).
```

As eg `deepvariant=1.4` is used in eg [snakemake-wrappers](https://github.com/snakemake/snakemake-wrappers/blob/master/bio/deepvariant/environment.yaml) this is problematic as it breaks running pipelines (see [issue](https://github.com/snakemake/snakemake-wrappers/issues/2296)).

Maybe this is related to issue #40448 (eg if `tensorflow` removed builds which could lead to this `etils` issue).

**Comment by aliciaaevans** [comment]
That recipe is pinned to Python 3.6 https://github.com/bioconda/bioconda-recipes/blob/3469952f4cdb24a923524498d3140588e8df45d5/recipes/deepvariant/meta.yaml#L37 which isn't included in the Bioconda pinnings any more. I'm not sure if there is an upstream reason, but it was mentioned in [the deepvariant issue](https://github.com/google/deepvariant/issues/736) that it should be work with Python 3.8. So the recipe probably needs to be updated to use Python 3.8.

**Comment by kdm9** [comment]
should be fixed in #45214
